hammer/tui.go

```diff
@@ -26,97 +26,120 @@ import (
 	"k8s.io/klog/v2"
 )
 
-func hostUI(ctx context.Context, hammer *Hammer) {
+type tuiController struct {
+	hammer     *Hammer
+	app        *tview.Application
+	statusView *tview.TextView
+	logView    *tview.TextView
+	helpView   *tview.TextView
+}
+
+func newController(h *Hammer) *tuiController {
+	c := tuiController{
+		hammer: h,
+		app:    tview.NewApplication(),
+	}
 	grid := tview.NewGrid()
 	grid.SetRows(5, 0, 10).SetColumns(0).SetBorders(true)
-	// Status box
+
+	// Top: status box
 	statusView := tview.NewTextView()
 	grid.AddItem(statusView, 0, 0, 1, 1, 0, 0, false)
-	// Log view box
+	c.statusView = statusView
+
+	// Middle: log view box
 	logView := tview.NewTextView()
 	logView.ScrollToEnd()
 	logView.SetMaxLines(10000)
 	grid.AddItem(logView, 1, 0, 1, 1, 0, 0, false)
+	c.logView = logView
+
+	// Bottom: help text
+	helpView := tview.NewTextView()
+	helpView.SetText("+/- to increase/decrease read load\n>/< to increase/decrease write load\nw/W to increase/decrease workers")
+	grid.AddItem(helpView, 2, 0, 1, 1, 0, 0, false)
+	c.helpView = helpView
```

Comment:
Is it necessary to create the local variables `statusView`, `logView` and `helpView`?

Reply:
I agree it's not necessary but it's also not harmful. Personally I like the separation of having the local variables be used for construction, and then assign when it's finished, but it doesn't matter. I'll leave it this way.